example/messaging/index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Messaging demo</title>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css"
+          integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
+    <style type="text/css">
+        ul {
+            list-style: none;
+            margin: 0;
+            padding: 0;
+        }
+
+        li {
+            margin: 0;
+            padding: 4px 2px;
+            border-bottom: 1px solid silver;
+            overflow: auto;
+        }
+        label {
+            width: 100%;
+        }
+        input {
+            width: 100%;
+        }
+    </style>
+</head>
+<body>
+<div class="container">
```

Comment:
Add some docs here. Where do I get the token and permission?

Reply:
These field are automatically populate by Firebase.
I show them to know the state of the app.
And you need the token to create a push notification from curl/postman/etc